.packit.yaml

```diff
@@ -129,71 +132,125 @@ jobs:
         # tag resources as sst_upgrades to enable cost metrics collection
         settings:
           provisioning:
-            post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys"
+            post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; yum-config-manager --enable rhel-7-server-rhui-optional-rpms"
             tags:
               BusinessUnit: sst_upgrades
+  env:
+    SOURCE_RELEASE: "7.9"
+    TARGET_RELEASE: "8.6"
+    RHUI: "aws"
+    LEAPPDATA_BRANCH: "upstream"
+
+# On-demand minimal beaker tests
+- &beaker-minimal-79to86
+  <<: *default-79to86
+  manual_trigger: True
+  labels:
+    - beaker-minimal
+    - beaker-minimal-7.9to8.6
+  identifier: tests-7.9to8.6-beaker-minimal
+  tmt_plan: "(?!.*max_sst)(.*tier1)(.*partitioning_monolithic|.*separate_var_usr_varlog|.*uefi|.*oamg4250_lvm_var_xfs_ftype0)"
+
+# On-demand kernel-rt tests
```

Comment:
Another nitpick, I would make this more unique, so you can search for sections easily without the knowledge of the identifiers/job handle names
```suggestion
# On-demand kernel-rt-79to86 tests
```

Reply:
That's more a // bridge comment for a bridge, I'd rather keep it as is, the most important part is that it clearly says this test is on demand.

Reply:
Sounds good